### DISCUSSION TITLE
Welcome to custom-stuff Discussions!

### DISCUSSION THREAD

**Discussion by JarvisCraft** [divine-craft/custom-stuff#36] [General]
## 👋 Welcome!

This is Discussions a place of Custom Stuff project aimed at providing better community experience for you!
Here you are free to:
 - Ask your questions about the API
 - Discuss and propose new features
 - Communicate with other developers
 - Share your own experience with using this project

**Comment by GoldenGidora** [comment]
![1-51581-128](https://user-images.githubusercontent.com/30473706/101673561-632f8300-3a68-11eb-80b7-461875ef853a.png)
Так а значит я ваш вот этот пендосский не понимаю, поэтому буду ждать русских в комментариях. Ставьте лайки, пж
